x/smartaccounts/spec/01_concepts.md

```
@@ -0,0 +1,77 @@
+Smart account extends the authentication capabilities of a base account.
+It assumes authentication methods added to base account, these authentication methods can be of type
+* Webauthn based
+* k256 based
+All of the authentication methods can be rotated if needed.
+
+## Limitations of Traditional Crypto Wallets
+
+Externally owned accounts (EOAs), or traditional crypto account, come with several drawbacks:
+
+- Single point of failure: One private key controls the entire account
+    - High risk: Losing your private key means losing access to your funds permanently
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)